buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -133,6 +133,18 @@ final class BuckFileGenerator {
             ":${rule.name}"
         }
 
+        // SqlDelight
+        if (target.sqldelight != null) {
+            String runner = target.sqldelight.externalDeps.find { it.contains("SqlDelightBin") }
+            if (runner == null) {
+                throw new IllegalStateException("SqlDelight model module must declare the SqlDelightBin dependency, " +
+                        "read more at https://github.com/OkBuilds/OkBuck/wiki/Known-caveats#sqldelight-work-around")
+            }
+            rules.add(GenSqlDelightRuleComposer.compose(target,
+                    target.project.rootProject.projectDir.absolutePath + "/" + runner))
```

Comment:
This seems to reach in too deep to an implementation detail `target.project.rootProject.projectDir.absolutePath + "/" + runner`
We should expose the path to the sqldeligght jar via the target api similar to the retrolamdba jar